pyseir/ensembles/ensemble_runner.py

```diff
@@ -94,27 +194,27 @@ def run_ensemble(self):
         Run an ensemble of models for each suppression policy nad generate the
         output report / results dataset.
         """
+        for suppression_policy_name, suppression_policy in self.suppression_policies.items():
+            logging.info(f'Running simulation ensemble for {self.county_metadata["county"]}, {self.county_metadata["state"]} {suppression_policy_name}')
 
-        for suppression_policy in self.suppression_policy:
-            logging.info(f'Generating For Policy {suppression_policy}')
-
-            parameter_ensemble = ParameterEnsembleGenerator(
+            parameter_sampler = ParameterEnsembleGenerator(
                 fips=self.fips,
                 N_samples=self.n_samples,
                 t_list=self.t_list,
-                suppression_policy=generate_empirical_distancing_policy(
-                    t_list=self.t_list,
-                    fips=self.fips,
-                    future_suppression=suppression_policy
-                )).sample_seir_parameters()
-
+                suppression_policy=suppression_policy)
+            parameter_ensemble = parameter_sampler.sample_seir_parameters(override_params=self.override_params)
```

Comment:
what is override_params?

Reply:
See the docstring for the consuming method. It overrides the parameter sampler values.

Reply:
This is temporary anyway since you are going to generate configs.